tasks/create.rb

```diff
@@ -8,13 +8,12 @@
 
 require_relative '../../ruby_task_helper/files/task_helper.rb'
 
-# This task creates objects
-class SnowCreate < TaskHelper
-  def task(table: 'incident',
+# This task creates records
+class ServiceNowCreate < TaskHelper
```

Comment:
We're definitely gonna need some better error handling here. Right now, this code swallows any API errors (because it doesn't check for the response's status, it just prints it as-is). It'd also be nice to DRY up some of this common "perform request" code, and avoid having to extract credentials all the time. That "perform request" code could do the error-checking.

Reply:
I'm wondering if we want all responses (200 or not) to go to stdout since the task did it's job to get that response properly. Then we would print any errors to stderr that actually happened inside the task.

Reply:
Moved the http code to a separate class, but still thinking about the best way to handle non 200s. Sent errors to STDERR.